breadcrumbs/src/layers/Scrollbar.js

```diff
@@ -71,15 +90,15 @@ export default class Scrollbar {
             this.p.line(
                 this.left,
                 _z,
-                this.width + this.left,
+                this.width + this.left + e.offset,
                 _z
             );
         });
 
         // Draw thumb:
         this.p.stroke(200);
         this.p.line(
-            this.left, this.top + this.height * (this.im.currentZ / this.im.nSlices),
+            this.left - 5, this.top + this.height * (this.im.currentZ / this.im.nSlices),
```

Comment:
@j6k4m8 will the offset here of "5" be the same as the one above? can you use the const HIGHLIGHT_OFFSET_AMOUNT ?

Reply:
Not the same — these are addressing different positioning artifacts. But also fine to make this a constant if you'd prefer it. It's only used once, though — and in a tight render loop.

Reply:
fine to leave as is then @j6k4m8